src/Components/Scenes/ViewRequests/RequestsInbox/RequestsInbox.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import RequestTableTitle from '../RequestTableTitle';
+import RequestTable from '../RequestTable';
+import { transformRequests } from '../../../../State/helpers';
+
+// Approver email is hard-coded until authentication is implemented
+const approverEmail = 'jonathan.delarosa@pnmac.com';
+const requestsInboxTableColumns = [
+  { name: 'Request', property: 'databasename' }, // placeholder from name property
+  { name: 'Description', property: 'description' },
+  { name: 'Status', property: 'requeststatus' },
+  { name: 'Date Requested', property: 'createddate' }
+];
+
+const RequestsInbox = props => {
+  const { requests, approverRequestsFetch } = props;
+
+  // Fetch all inbound requests given for approver's email
+  useEffect(() => {
+    approverRequestsFetch(approverEmail);
+  }, [approverRequestsFetch]);
+
+  const reqs = transformRequests(requests);
+
+  const setFooterButtonText = selected =>
```

Comment:
I'm realizing these two-line one-liners are probably just because of `prettier`, so maybe not worth being picky over. My b